documentation/smg/06_deployment_process.md

```diff
@@ -0,0 +1,79 @@
+# Distributing Horace to users
+
+Current Horace distribution process uses two channels, which allows us frequently modify the code,
+satisfy the requests of different groups of users and rapidly react to the bugs, encountered by users
+running their experiments in ISIS. The distribution is deployed  
```

Comment:
unnecessary new line, breaks the text in the formatted document.

Reply:
I've reformated it but my MD editor was showing and still shows smooth formatting.